io/plugins/eu.esdihumboldt.hale.io.gml.ui/src/eu/esdihumboldt/hale/io/gml/ui/GmlWriterSettingsPage.java

```diff
@@ -131,54 +149,164 @@ protected void createContent(Composite page) {
 		Group writeFormat = new Group(page, SWT.NONE);
 		GridLayoutFactory.swtDefaults().numColumns(2).applyTo(writeFormat);
 		GridDataFactory.fillDefaults().grab(true, false).applyTo(writeFormat);
-		writeFormat.setText("Formatted Number");
+		writeFormat.setText("Output formatting");
 
-		checkNumberFormat = new Button(writeFormat, SWT.CHECK);
-		checkNumberFormat.setText("Use a formatted number output for geometry coordinates");
-		GridDataFactory.swtDefaults().span(2, 1).applyTo(checkNumberFormat);
-		checkNumberFormat.addSelectionListener(new SelectionAdapter() {
+		enableCoordinateFormat = new Button(writeFormat, SWT.CHECK);
+		enableCoordinateFormat.setText("Use a formatted number output for geometry coordinates");
```

Comment:
While we are at it, should we add a warning that this may break the geometries by introducing duplicate points? The formatting will not "fix" a geometry nor warn about geometries that become invalid (e.g. a polygon that ha less than three distinct points).

Reply:
As a side note, the problem w/ duplicate points is also addressed in #398